features/intersection-observer-v2.yml

```diff
@@ -1,6 +1,6 @@
 name: Intersection observer visibility tracking
 description: The `trackVisibility` parameter for the `IntersectionObserver` constructor enables tracking the visibility of an element, to detect if it may be obscured by other content or visual effects. Also known as IntersectionObserver v2.
-spec: https://w3c.github.io/IntersectionObserver/v2/
+spec: https://w3c.github.io/IntersectionObserver/#calculate-visibility-algo
```

Comment:
@foolip the v2 spec merged into the Intersection Observer spec. Is this new URL choice OK?

https://github.com/w3c/browser-specs/pull/1210
https://github.com/w3c/IntersectionObserver/pull/523

Reply:
Yep, that's OK I think. The other option would be to point to one of these:

- https://w3c.github.io/IntersectionObserver/#dom-intersectionobserver-trackvisibility
- https://w3c.github.io/IntersectionObserver/#dom-intersectionobserverinit-trackvisibility
- https://w3c.github.io/IntersectionObserver/#dom-intersectionobserverentry-isvisible

But that section header is fine.